packages/terra-collapsible-menu-view/src/_CollapsibleMenu.jsx

```diff
@@ -25,6 +25,11 @@ const propTypes = {
    * Button to display the menu from
    */
   button: PropTypes.element.isRequired,
+
+  /**
+   * Callback function that is called when the menu's selection state changes
+   */
+  onChange: PropTypes.func,
```

Comment:
I may be missing it, but where is this onChange prop used?

Reply:
You are right, I don't need this. The changes made in _CollapsibleMenu.jsx is to wrap onChange so that it can return the selectedKey (instead of the index from terra-list). However, it doesn't cover the workflow when a selection is made from the collapsed menu item. I've since remove all the changes in _CollapsibleMenu.jsx and handle the onChange in CollapsibleMenuViewItemGroup.